index.js

```diff
@@ -73,10 +73,16 @@ function Slideout(options) {
   this._fx = options.fx || 'ease';
   this._duration = parseInt(options.duration, 10) || 300;
   this._tolerance = parseInt(options.tolerance, 10) || 70;
-  this._padding = parseInt(options.padding, 10) || 256;
+  this._padding = this._translateTo = parseInt(options.padding, 10) || 256;
+  this._side = options.side || 'left';
+
+  // Change translateTo orientation on right side
+  if (this._side === 'right') {
```

Comment:
Using `this._orientation` we can rewrite the if block as follows:

```this._translateTo *= this._orientation;```

removing the if construction.